_pages/index.md

```diff
@@ -3,56 +3,15 @@ title: Your guide to launching software at 18F
-    * [CFPB's open source checklist](https://github.com/cfpb/open-source-project-template/blob/master/opensource-checklist.md)
-    * _Ask #wg-opensource if you have questions._
-* Make sure you have all the social media metadata and preview images.
-
-### Security and operations
+### Key roles
 
-* Review the [**security best practices**](security/), including [scanning](security/scanning/) and the [framework-specific guides](security/frameworks/).
-* Is [SSL](https://github.com/18f/https) enabled for everything?
-    * Do all of your URLs start with `https://`?
-    * Does visiting your site with `http://` redirect to `https://`?
-* Review the [**infrastructure best practices**](infrastructure/).
-* Reach out to the #infrastructure team at least a month in advance of launch to give them a heads up.
+- **Authorizing Official (AO)** The AO is ultimately responsible for determining if the risk of operating the system is acceptable, and if so, issuing an Authority to Operate (ATO) for that system. At 18F the Director of Infrastructure is the AO.
+- **System Owner** The system owner is usually the product lead or tech lead of the project team. They will be named in the ATO documents and are the main contact during the evaluation process that leads up to an ATO.
 
-### Also consider
-
-* Do you have a metrics and measurements strategy? Who owns this?
-* Review the [18F Guides](https://guides.18f.gov).
-* Once you have reviewed everything here, the DigitalGov team at GSA has collected a list of [Requirements for Federal Websites and Digital Services](http://www.digitalgov.gov/resources/checklist-of-requirements-for-federal-digital-services/) that you should familiarize yourself with. We recommend reviewing this last, because many of the requirements are explained or managed in the above guides and processes.
-
-### Post-launch releases
-
-For every release after your public launch, you should consider: _(Explained in detail above.)_
-
-* Accessibility reviews should be done for any changes.
-* Do you need to update your GitHub documentation, or your `.about.yml`?
-
----
+### ATO status of Federalist/Pages sites
 
-How'd it go? Is this list missing anything important? [Submit a pull request](https://github.com/18f/before-you-ship) or [open an issue](https://github.com/18f/before-you-ship/issues/new)!
+If you are publishing a new site through [Federalist](infrastructure/#federalist) or [18F Pages](infrastructure/#f-pages) and it's not connecting to any APIs or third-party services (i.e. it's a simple static site), the site is considered part of that system, so it does not require its own ATO ([source](https://github.com/18F/before-you-ship/issues/95#issuecomment-174011747)). *Note: Technically, static site builders are just adding a collection of pages in an existing system. Therefore, from an ATO perspective, "sites" created through Federalist or 18F Pages remain within the respective security boundary, and thus ATO.*
```

Comment:
Hmm. This page doesn't really flow into the other sections like it used to 😕  The bottom two sections feel a bit disjointed, as well. We may need some content design help here.